challenges-14-plus.yml

```
@@ -0,0 +1,472 @@
+asset-tracking:
+  question: |
+    I just came from our company meeting and I have big news! We have just released a
+    new version of our main CSS file!
+    
+    But I'm getting complaints from users who are unable to see those changes! Oh no!
+    
+    How can I make Turbo Drive show the new version of my site to all my users without
+    waiting for them to do a full refresh (i.e. by closing their browsers and come back
+    the next day)?
+  explanation: |
+    One of the few instances in which we can say that a full reload is *needed* is when
+    our front end code changes (JavaScript or CSS)!
+    
+    By adding file versioning to our system, we can make those files be downloaded once
+    they change. But this alone is not enough! Webpack prevents similar named files with
+    different hashes from running! And besides, if they did run, we would be duplicating
+    them on our `<head>` element.
+    
+    By leveraging the attribute `data-turbo-track="reload"` in our `<script>` and `<link>
+    tags, we can make Turbo Drive force a reload of the page if the collection of files
+    in the `<head>` element ever changes! This combination of techniques is our best
+    friend!
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          By modifying the file `config/packages/webpack_encore.yaml` and add the attribute:
+  
+          ```yml
+          'data-turbo-track': reload
+          ```
+
+          To both `link_attributes` and `script_attributes`.
+        b: |
+          By making sure you enable versioning in Webpack Encore:
+          
+          ```javascript
+          .enableVersioning(Encore.isProduction())
+          ```
+          
+          And by modifying the file `config/packages/webpack_encore.yaml` and add the attribute:
+          
+          ```yml
+          'data-turbo-track': reload
+          ```
+          
+          To both `link_attributes` and `script_attributes`.
+        c: |
+          By modifying the file `config/packages/webpack_encore.yaml` and add the attribute:
+  
+          ```yml
+          'data-turbo-track': refresh
+          ```
+
+          To `script_attributes`.
```

```suggestion
          To `link_attributes`.
```

(makes it more realistic since we're talking about CSS)